modules/virt-listing-vmis-web.adoc

```diff
@@ -0,0 +1,16 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis-web.adoc
+
+[id="virt-listing-vmis-web_{context}"]
+= Listing standalone virtual machine instances using the web console
+
+Using the web console, you can list and view standalone virtual machine instances (VMIs) in your cluster that are not owned by virtual machines (VMs).
```

Comment:
IMHO it will be nice also to add a note at the buttom that that emphasise that the list in the cli include all the VMIs while the list in the web console will show only the standalone VMIs.

Reply:
+1

Reply:
Added the note:
https://cnv-3703-vmis--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-manage-vmis.html#virt-listing-vmis-web_virt-manage-vmis

Please review and advise if this PR can move to QE review. Thanks!

Reply:
LGTM!

Reply:
lgtm too :-)

Reply:
Thanks for the LGTM status. This PR is ready for QE Review.